.github/workflows/ci.yml

```diff
@@ -148,13 +144,15 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        node_version: [18]
+        node_version: ['lts/*']
```

Comment:
It seems inconsistent that we're using `lts/*` directly here and `env.NODE_VERSION` elsewhere. I'd be fine with removing the variable entirely and using `lts/*` everywhere, but we should be consistent.

Reply:
Done.